handlers/handlers.go

```diff
@@ -300,8 +300,38 @@ func HandleNoteApiRequest(
 
 		fmt.Fprint(responseWriter, string(noteString))
 
+	case http.MethodDelete:
+
+		id, err := strconv.ParseInt(request.URL.Query().Get("id"), 10, 64)
```

Comment:
We talked about this over slack, but I think "proper" rest api design is to have this in the path, not in the query param (`DELETE /notes/:id` instead of `DELETE /notes/?id=:id`)

Reply:
or `DELETE /user/:user-id/notes/:note-id` if note is a nested resource of user